tests/unit/test_app.py

```diff
@@ -715,3 +738,21 @@ def test_can_serialize_custom_authorizer():
             'authorizerResultTtlInSeconds': 10,
         }
     }
+
+
+class TestCORSConfig(object):
```

Comment:
Can you add a test where `CORSConfig` objects are equal with non-default params? (add  `allow_headers,expose_header,max_age,allow_credentials`).